test/data/ReqMgr/requests/Integration/SC_Nano.json

```diff
@@ -59,8 +59,6 @@
             "CMSSWVersion": "CMSSW_10_6_26",
             "Campaign": "RunIIAutumn18NanoAODv5",
             "ConfigCacheID": "0fd238f99b39e8fef9f658d1dd1d6f61",
-            "EventsPerJob": 40000,
```

Comment:
@amaltaro , is it intended the removal of these two keys? The updated description says "Job splitting hardcoded to 10k EpJ and 10k EpL", but I'm failing to find where this is actually hardcoded.

Reply:
Thanks! We could remove both and rely on the automatic 8h-long job splitting (based on TimePerEvent). However, let me force this workflow to a given number of events per job (lumi size cannot be modified, as it has input dataset).